geozero/src/postgis/postgis_sqlx.rs

```diff
@@ -144,7 +144,8 @@ macro_rules! impl_sqlx_postgis_encode {
             fn encode_by_ref(
                 &self,
                 buf: &mut sqlx::postgres::PgArgumentBuffer,
-            ) -> sqlx::encode::IsNull {
+            ) -> std::result::Result<IsNull, Box<(dyn std::error::Error + Send + Sync + 'static)>>
```

Comment:
so my understanding here is that sqlx 0.8 modified their Encode trait to handle boxed errors? If so, seems reasonable

Reply:
I update this to qualified use